dash/long_callback/managers/diskcache_manager.py

```diff
@@ -0,0 +1,117 @@
+import platform
+from . import BaseLongCallbackManager
+
+
+class DiskcacheLongCallbackManager(BaseLongCallbackManager):
+    def __init__(self, cache, cache_by=None, expire=None):
+        import diskcache  # pylint: disable=import-outside-toplevel
+
+        if not isinstance(cache, diskcache.Cache):
+            raise ValueError("First argument must be a diskcache.Cache object")
+        super().__init__(cache_by)
+
+        # Handle process class import
+        if platform.system() == "Windows":
```

Comment:
@LiamConnors Can you try change this to `if True:` and install the `multiprocess` library and see if that makes a difference?

Reply:
Thanks @jonmmease. Yep, that worked.

Reply:
Great! I went ahead and made this change, so all platforms will use this logic path.